.github/workflows/spellbook.yml

```diff
@@ -0,0 +1,22 @@
+name: DBT Spellbook run
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+      - 'scripts/**'
+      - 'Pipfile'
+      - '.gitignore'
+      - 'tokens/**'
```

Comment:
Are these necessary for the spellbook subproject?

Reply:
I think we may also need `macros` and `sources`

Reply:
This is the main existing one and this is an ignore list, so we ignore the subprojects only here.